packages/react-dom/src/events/__tests__/DOMPluginEventSystem-test.internal.js

```diff
@@ -679,25 +679,33 @@ describe('DOMPluginEventSystem', () => {
             );
           }
 
-          ReactDOM.render(<Parent />, container);
+          await act(async () => {
+            ReactDOM.render(<Parent />, container);
+          });
 
           const parent = container.lastChild;
           expect(parent.id).toEqual('parent');
-          dispatchClickEvent(parent);
+
+          await act(async () => {
+            dispatchClickEvent(parent);
+          });
 
           expect(log).toEqual(['parent']);
 
           const child = parent.lastChild;
           expect(child.id).toEqual('child');
-          dispatchClickEvent(child);
+
+          await act(async () => {
+            dispatchClickEvent(child);
```

Comment:
I had to change these tests because they were previously relying (accidentally) on the click event to flush the pending passive effects. Which after this change, they no longer do.

Fortunately, the fix is pretty easy. Unfortunately, this change affects legacy mode, too. I'm curious how many tests would be affected in normal practice.

Reply:
Hm. Yea that might be a big one.